crates/solver/src/settlement.rs

```diff
@@ -502,6 +502,17 @@ impl Settlement {
             None
         }
     }
+
+    pub fn objective_value(
```

Comment:
nit: I'd leave this logic into the boundary (to avoid adding new code to the legacy codebase).

Reply:
fine with that, removed